profiles.md

```diff
@@ -0,0 +1,235 @@
+        "language": null,
+        "authority": null,
+        "confidence": -1,
+        "place": 0
+      }]
+  },
+  "netid": "msmith@samltest.id",
+  "lastActive": "2019-10-25T08:05:46.530+0000",
+  "canLogIn": true,
+  "email": "msmith@samltest.id",
+  "requireCertificate": false,
+  "selfRegistered": false,
+  "type": "eperson"
+}
+```
+
+It would respond with:
+* 200 OK - Returning the eperson if there's a match
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions to view the eperson
+* 404 Not found - if the eperson doesn't exist
+
+## Creating a new Profile
+**POST /api/eperson/profiles?eperson=<:eperson-uuid>**
+
+Create a new profile for the specified person or for the current user. This action can only be done by the EPerson themselves or an Administrator.
+
+Parameters:
+* eperson-uuid (optional): the uuid of eperson for which the profile is to be created; if not specified, the eperson is the authenticated one
+
+Status codes:
+* 201 Created - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions
+* 409 Conflict - if for the specified eperson a profile already exists
+* 422 Unprocessable Entity - if the specified eperson does not exist
+
+In case of response with status 201 the endpoint returns the newly created resource, while in case of 409 it returns the resource that generated the conflict.
+
+To create a new profile use 
+```bash
+curl -i -X POST ${dspace-url}/api/eperson/profiles?eperson=eb645ef8-1373-41eb-bf67-6afcea7e2069 -H "Content-Type:application/json"
+```
+
+## Claim an existing Profile
+**POST /api/eperson/profiles?eperson=<:eperson-uuid>**
+
+Modify the profile item specified in the request content to make the given eperson its owner. The content-type is uri-list with the url of the item's profile to be claimed.
+This action can only be done by the EPerson themselves or an Administrator.
+
+Parameters:
+* eperson-uuid (optional): the uuid of eperson for which the profile is to be claimed; if not specified, the eperson is the authenticated one
+
+Status codes:
+* 201 Created - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions
+* 409 Conflict - if the specified eperson has already a profile or if the profile to be claimed is already owned
+* 404 Unprocessable Entity - if the specified eperson does not exist
+
+An example curl call:
+```bash
+ curl -i -X POST ${dspace-url}/api/eperson/profiles?eperson=eb645ef8-1373-41eb-bf67-6afcea7e2069 -H "Content-Type:text/uri-list" \
+ --data ${dspace-url}/api/core/items/cec6ee1b-7730-44da-a224-a7c5af63f821
+```
+
+## Hide/unhide a profile
+**PATCH /api/eperson/profiles/<:eperson-uuid>**
+
+This operation allow to change the visibility of one profile. This action can only be done by the EPerson themselves or an Administrator.
```

Comment:
We should update this description to define what we mean by "visibility".  A value of true means the Profile is given Anonymous READ access.  A value of false, means the Profile is not accessible Anonymously.